src/+otp/+brusselator/BrusselatorParameters.m

```diff
@@ -1,9 +1,10 @@
 classdef BrusselatorParameters
-    %BRUSSELATORPARAMETERS 
+    %BRUSSELATORPARAMETERS Parameters for the Brusselator problem
+    %   See also otp.brusselator.BrusselatorProblem
     properties
-        %A is a combination of an excess reactant and reaction rate
+        %A concentration of excess reactant A in the Brusselator reaction
```

Comment:
this is not true, because it also represents the reaction rate in one of the directions.

The way the problem is discretized and simplified, the reaction rate and concentration of excess reactant are combined into one dimensionless constant.

same comment for B